cmake/FindNUMPY.cmake

```diff
@@ -39,11 +39,7 @@
 #============================================================================
 
 # Finding NumPy involves calling the Python interpreter
-if(NumPy_FIND_REQUIRED)
-    find_package(PythonInterp REQUIRED)
-else()
-    find_package(PythonInterp)
-endif()
+find_package(PythonInterp REQUIRED)
```

Comment:
maybe you are right there, you don't want to fail when numpy is not a requirement.